docker/pypi/wmagent-couchdb/couchdb-docker-run.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+### This script is to be used for running the Couchdb docker container at a VM
+### Its sole purpose is to set all the needed mount points from the Host VM and
+### forward all Couchdb runtime parameters to the Couchdb container entrypoint run.sh
+### It accepts only the set of parameters relevant to Couchdb's container run.sh
+### and no build dependent ones. The docker image tag to be searched for execution is
+### always `latest`.
+
+
+# NOTE: In the help call to the current scrit we only repeat the help and usage
+#       information for all the parameters accepted by run.sh.
+help(){
+    echo -e $*
+    cat <<EOF
+
+The script to be used for running a Couchdb docker container at a VM. The full set of arguments
+passed to the current script are to be forwarded to the Couchdb container entrypoint 'run.sh'
+
+Usage: couchdb-docker-run.sh [-t <team_name>] [-n <agent_number>] [-f <db_flavour>]
```

Comment:
I don't see why team name, agent number and db flavour would be relevant to CouchDB service(?)

Reply:
they are not. This  message is a left over from an old version which was used in the WMAgent container.